react/src/index.tsx

```diff
@@ -14,8 +14,7 @@ import { theme } from "./utils/theme";
 
 if (process.env.NODE_ENV === "development") {
```

Comment:
Can you delete this entire block? It's stale code and the GraphQL mock query might not even work